src/api/types/cert-manager.ts

```diff
@@ -21,7 +23,7 @@ export interface V1CertificateSpec {
   secretName: string
   issuerRef: V1CertificateSpecIssuerReference
   commonName: string
-  dnsNames: List<string>
+  dnsNames: string[]
```

Comment:
I'm not sure where the `List` comes from and if it can be replaced with `Array`. I suppose it should be OK as far as the build doesn't fail.